examples/dapperish/dapper.go

```diff
@@ -9,8 +9,15 @@ import (
 	"sync"
 
 	"github.com/opentracing/api-golang/opentracing"
+	"github.com/opentracing/api-golang/opentracing/standardtracer"
 )
 
+func NewDapperishTracer(processName string) opentracing.OpenTracer {
```

Comment:
I think golint might complain about stutter on this one. You probably want just this: `dapperish.NewTracer()`